src/jaegertracing/TraceID.h

```diff
@@ -43,13 +43,8 @@ class TraceID {
     template <typename Stream>
     void print(Stream& out) const
     {
-        if (_high == 0) {
-            out << std::hex << _low;
-        }
-        else {
-            out << std::hex << _high << std::setw(16) << std::setfill('0')
-                << std::hex << _low;
-        }
+        out << std::setw(16) << std::setfill('0') << std::hex << _high
```

Comment:
this does not seem backwards compatible, what's the reason for removing `if (_high == 0) {`?

Reply:
done

Reply:
I though the trace id is now always 32 hex chars long. But I was wrong.

Reply:
No, all Jaeger SDKs still use 64bit by default, you have to explicitly enable 128bit.

Reply:
Yes, noticed that after your comment. I misread the prs for the other sdks.